roles/collect_modified_images/tasks/parse_image_names.yml

```diff
@@ -0,0 +1,9 @@
+- name: "split item"
+  set_fact:
+    timestamp: "{{ item.split(',')[0] }}"
+    image: "{{ item.split(',')[1][15:-1] }}"
+
+- name: "parse and return image name when timestamp is after start_time"
+  set_fact:
+    pulled_images: "{{ pulled_images + [image] }}"
+  when: "{{ timestamp > deployment_start_time }}"
```

Comment:
I couldn't see where `deployment_start_time` is initialized or passed previous to importing these tasks.

Reply:
It's in local-test-operator.yml when the role is called.  I also added it to `roles/collect_modified_images/defaults.main.yml` just in case